### PR TITLE
Feature/specification

### PR DESCRIPTION
**Pull request by anamaiabap**
Trocando a regra de ativação do campo especificação 
De:
Captura de Tela 2021-08-18 às 16 42 52
Para: 
Captura de Tela 2021-08-18 às 16 39 45

E consertando errinhos na parte de salvar e editar

### CODE REVIEW

File: react/Components/imageEdit.tsx [hunk 1]
```diff
@@ -51,14 +51,18 @@ const ImageEdit: FC = () => {
...
+          <Button
+            onClick={(e: SyntheticEvent) => {
+              e.preventDefault()
+              provider.setShowImage(true)
+            }}
+          >
+            {intl.formatMessage(commonModal.cancel)}
+          </Button>
+        ) : (
+          ''
```
Comment (pscruzzz) [inline react/Components/imageEdit.tsx:64]: How about having an empty tag?
```suggestion
          <></>
```

File: react/utils/autoComplete.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ const AutoComplete: FC<Props> = ({
 }: {
   onChange: any
   name: string
-  value: string
+  value: any
```
Comment (pscruzzz) [inline react/utils/autoComplete.tsx:30]: Could't this be an object referencing the `id`? Something like:
`
value: {id: idType}
`

Where idType could be number, string or etc.

File: react/Components/imageEdit.tsx (hunk 1 above)
Comment (wagnerlduarte) [reply react/Components/imageEdit.tsx:64]: In this case I think is a better idea to use the `&&` operator!

```
        {provider.file.result && (
          <Button
            onClick={(e: SyntheticEvent) => {
              e.preventDefault()
              provider.setShowImage(true)
            }}
          >
            {intl.formatMessage(commonModal.cancel)}
          </Button>
        )
```

Comment (pscruzzz) [reply react/Components/imageEdit.tsx:64]: That's better!!

**Review by wagnerlduarte — APPROVED** [review]

**Review by wagnerlduarte — APPROVED** [review]

**Review by pscruzzz — APPROVED** [review]

**Pull request merged**